localisation/00_zuwc_united_localisation.csv

```diff
@@ -3057,6 +3057,14 @@ RULE_MISTS_OF_PANDARIA_ON;On;;;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_ON_DESC;The Mists of Pandaria are enabled, making Pandaria isolated till they're gone around the 613th year.;;;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_OFF;Off;;;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_OFF_DESC;The Mists of Pandaria are disabled thus Pandaria isn't isolated.;;;;;;;;;;;;;x
+emperor_of_pandaria;Blood of [This.GetFirstNameWithNick];;;;;;;;;;;;;x
+emperor_of_pandaria_desc;The next emperor of Pandaria after legendary Shaohao, [This.GetFirstNameWithNick] restored imperial traditions and united Pandaria once again.;;;;;;;;;;;;;x
+form_the_pandaria_empire;Restore the Pandaria Empire;;;;;;;;;;;;;x
+form_the_pandaria_empire_desc;You become the next emperor of Pandaria after legendary Shaohao.;;;;;;;;;;;;;x
+EVTTITLE_WCPAN_200;[e_pandaria_empire.GetName];;;;;;;;;;;;;x
+EVTDESC_WCPAN_200;Ten thousand years ago, the last emperor of Pandaria, Shaohao, became one with the land and disappeared. His last breath became the mist that veiled Pandaria for ages. After the emperor disappeared, the power in Pandaria shifted to the local level. With all threats walled off, there was no longer a need for an emperor anymore... but times change.\n\n[target_emperor.GetTitledFirstName] belives that Pandaria needs the emperor�s strong hand once again. Controlling most of Pandaria, [target_emperor.GetSheHe] declares [target_emperor.GetHerselfHimself] the next emperor after Shaohao.;;;;;;;;;;;;;;x
+EVTOPTA_WCPAN_200;I'm the hearth of Pandaria!;;;;;;;;;;;;;x
```

Comment:
Just my opinion, but would `heart` not be better here than `hearth`?
```suggestion
EVTOPTA_WCPAN_200;I'm the heart of Pandaria!;;;;;;;;;;;;;x
```

Reply:
Done in c1b4853ae2a70fe922a78be01fa3cbe020564096.